src/openrct2/peep/Peep.h

```diff
@@ -418,7 +418,7 @@ enum class PeepNauseaTolerance : uint8_t
     High
 };
 
-enum PeepItem
+enum class PeepItem : uint32_t
 {
     // ItemStandardFlags
     PEEP_ITEM_BALLOON = (1 << 0),
```

Comment:
The naming should be changed as well to adhere the other enums, also the elements should be not defined as flags which also means all code needs to be updated using EnumToFlags accordingly.

Reply:
Renaming is done as well as flags are removed from this enum. It's now only uint32_t. And here is my doubt -  is that correct type for this enum? Maybe uint8_t would be better?